ql/src/java/org/apache/hadoop/hive/ql/reexec/ReExecDriver.java

```diff
@@ -83,6 +83,11 @@ public int compile(String command, boolean resetTaskIds) {
     return coreDriver.compile(command, resetTaskIds);
   }
 
+  @VisibleForTesting
+  public List<IReExecutionPlugin> getPlugins() {
```

Comment:
could it be package private if we move the testt under reexec package?

Reply:
I think we cannot make a package private by moving it.